src/index.html

```diff
@@ -7,8 +7,54 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="index.html" class="logo">
+        <img src="images/logo.png" alt="header_Logo">
```

Comment:
'alt' should describe the content of an image. We can specify that this is the logo of `Moyo` company. 
```suggestion
        <img src="images/logo.png" alt="Moyo logo">
```